p2p_wallet/Scenes/Main/ChooseWallet/ChooseWallet.ViewController.swift

```diff
@@ -8,45 +8,46 @@
 import Foundation
 import UIKit
 import BECollectionView
+import BEPureLayout
 
 extension ChooseWallet {
-    class ViewController: WLIndicatorModalVC {
+    final class ViewController: BaseVC {
         
         // MARK: - Properties
         private let viewModel: ViewModel
+
         private lazy var rootView: RootView = {
             let rootView = RootView(viewModel: viewModel)
             rootView.forwardDelegate(to: self)
             return rootView
         }()
-        
+
         // MARK: - Initializer
-        init(viewModel: ViewModel)
-        {
+        init(title: String?, viewModel: ViewModel) {
             self.viewModel = viewModel
+
             super.init()
+
+            self.title = title ?? L10n.selectToken
```

Comment:
- When select source token, the text has to be `Select the first token`
![Simulator Screen Shot - iPhone 11 - 2021-12-15 at 17 35 19](https://user-images.githubusercontent.com/6975538/146171077-38e598c4-8413-4f98-b7bf-70b4a7e808cd.png)